.github/workflows/ci.yml

```diff
@@ -0,0 +1,64 @@
+name: Python package
+
+on: [push, pull_request]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+        - 3.6
+        - 3.7
+        - 3.8
+        - 3.9
+    name: Tests for Python ${{ matrix.python-version }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: conda-incubator/setup-miniconda@v2
+      with:
+        auto-update-conda: true
+        python-version: ${{ matrix.python-version }}
+        activate-environment: formulate-env
+
+    - name: Install ROOT
+      shell: "bash -l {0}"
```

Comment:
You can set this as the default shell above (for the job), and save just a tiny bit of cruft here. Not major.

Reply:
(I think GHA added default shell settings pretty much for this Action, by the way :) )

Reply:
I thought so as well, but then it didn't work. Maybe was something different, I'll retry

Thanks for the default, I wasn't aware of this feature!

Reply:
https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_iddefaultsrun

So you can add one to the job.

Reply:
```yaml
defaults:
      run:
        shell: "bash -l {0}"
```

Should work.

Reply:
Yep, I return to that, I had it before and tried it out, but yep, fails without the console